__tests__/api/health.spec.js

```diff
@@ -16,7 +16,11 @@
 //
 
 import { default as request } from 'supertest'; // eslint-disable-line
-import app from '../../src';
+import expressApp from '../../src';
+import mockAuth from '../../src/libs/mock.authmware';
+
+
+const app = expressApp(mockAuth, passport => passport.authenticate('mock'));
```

Comment:
I don't think this is necessary. The test was working just fine and Jest jest can automatically mock objects if you setup the test structure as needed.